tests/libnet/ipv6.go

```diff
@@ -0,0 +1,57 @@
+package libnet
+
+import (
+	"time"
+
+	netutils "k8s.io/utils/net"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/log"
+	"kubevirt.io/kubevirt/tests/console"
+	"kubevirt.io/kubevirt/tests/libvmi"
+)
+
+func ConfigureIPv6OnVMI(vmi *v1.VirtualMachineInstance) error {
+
+	hasEth0Iface := func() bool {
+		err := console.RunCommand(vmi, "ip a | grep -q eth0", 30*time.Second)
+		return err == nil
+	}
+
+	hasGlobalIPv6 := func() bool {
+		err := console.RunCommand(vmi, "ip -6 address show dev eth0 scope global | grep -q inet6", 30*time.Second)
+		return err == nil
+	}
+
+	clusterSupportsIpv6 := func() bool {
```

Comment:
If we are already doing refactoring here-. We can use `libnet.IsClusterDualStack` instead of reading the ip from the pod.